docs.yaml

```diff
@@ -37,6 +37,8 @@ nanoc_config:
       exclude:
         - '^https?://travis-ci.org/*'
         - '^https?://ci.appveyor.com/*'
+        - 'https://twitter.com/datastax'
+        - 'http://sourceforge.net/projects/boost/files/boost-binaries/1.[0-9]+.0/boost_1_[0-9]+_0-msvc-[0-9]+.[0-9]+-(64|32).exe/download'
```

Comment:
I'd argue that instead of excluding this we should make anything matching this expression a redirect to https://www.boost.org/users/download/.  Whatever is using this link in the C++ docs is historical (there's no reference to it in the current docs) but it's pretty clear from the URL itself that we're looking for the download server for the Boost libs... and the URL I mention above is where you go to get those libs nowadays.